### PR TITLE
fix(build): Change publish-build-artifacts.sh to works with new packaging system

### PR DESCRIPTION
**Pull request by vikerman**

### CODE REVIEW

File: scripts/publish/publish-build-artifacts.sh [hunk 1]
```diff
@@ -38,40 +42,25 @@ function publishRepo {
     git config user.email "${COMMITTER_USER_EMAIL}" && \
     git add --all && \
     git commit -m "${COMMIT_MSG}" && \
-    git push origin $BUILD_BRANCH && \
-    git tag "2.0.0-build.${SHORT_SHA}.${LANG}" && \
+    git push origin master && \
+    git tag "2.0.0-build.${SHORT_SHA}" && \
```
Comment (IgorMinar) [inline scripts/publish/publish-build-artifacts.sh]: let's remove the `build.` from here.

File: scripts/publish/publish-build-artifacts.sh [hunk 2]
```diff
@@ -38,40 +42,25 @@ function publishRepo {
...
     git push origin --tags
   )
 }
 
+COMPONENT="core"
```
Comment (IgorMinar) [inline scripts/publish/publish-build-artifacts.sh]: we should loop over `packages-dist` and do this for every package in there.

**Comment by IgorMinar** [comment]
otherwise lgtm. I created all the repos you'll need. please go ahead and update the scripts.

**Comment by vikerman** [comment]
Merged into master d414734aac5d2d0778e1c251a52e3b671443a561

**Pull request closed without merging**